test/integration/org/controls/gcp.rb

```diff
@@ -0,0 +1,357 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+common_folder_name = attribute('common_folder_name')
+org_audit_logs_project_id = attribute('org_audit_logs_project_id')
+org_billing_logs_project_id = attribute('org_billing_logs_project_id')
+org_secrets_project_id = attribute('org_secrets_project_id')
+interconnect_project_id = attribute('interconnect_project_id')
+scc_notifications_project_id = attribute('scc_notifications_project_id')
+dns_hub_project_id = attribute('dns_hub_project_id')
+parent_resource_type = attribute('parent_resource_type')
+parent_resource_id = attribute('parent_resource_id')
+domains_to_allow = attribute('domains_to_allow')
+
+bigquery_logs_datasets = [
+  'activity_logs',
+  'system_event',
+  'data_access',
+  'vpc_flow',
+  'firewall_rules',
+  'access_transparency'
+]
+
+dns_hub_apis = [
+  'compute.googleapis.com',
+  'dns.googleapis.com',
+  'servicenetworking.googleapis.com',
+  'logging.googleapis.com',
+  'cloudresourcemanager.googleapis.com'
+]
+
+scc_notifications_apis = [
+  'logging.googleapis.com',
+  'pubsub.googleapis.com',
+  'securitycenter.googleapis.com'
+]
+
+org_secrets_apis = [
+  'logging.googleapis.com',
+  'secretmanager.googleapis.com'
+]
+
+logs_apis = [
+  'logging.googleapis.com',
+  'bigquery.googleapis.com'
+]
+
+boolean_policy_constraints = [
+  'constraints/compute.disableNestedVirtualization',
+  'constraints/compute.disableSerialPortAccess',
+  'constraints/compute.disableGuestAttributesAccess',
+  'constraints/compute.vmExternalIpAccess',
+  'constraints/compute.skipDefaultNetworkCreation',
+  'constraints/compute.restrictXpnProjectLienRemoval',
+  'constraints/sql.restrictPublicIp',
+  'constraints/iam.disableServiceAccountKeyCreation',
+  'constraints/storage.uniformBucketLevelAccess'
+]
+
+control 'gcp' do
+  title 'gcp step 1-org tests'
```

Comment:
nit: it might be worth breaking these up into separate controls for readability of both code and test output. For example you might have one for org projects, org logging and org policy

Reply:
Done. Thanks for the suggestion.